Sift/Core/WebhookValidator.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Sift.Core
+{
+    public class WebhookValidator
+    {
+        private const string Sha1Prefix = "sha1=";
+        public bool IsValidWebhook(String requestBody, String secretKey, string signatureWithPrefix)
+        {
+            if (IsWebHookAllowed(requestBody, secretKey, signatureWithPrefix))
```

Comment:
I think this `if` block could be replaced by just `return IsWebHookAllowed(requestBody, secretKey, signatureWithPrefix)`.

Although more broadly speaking, `IsValidWebhook` doesn't seem to do anything but delegate directly to `IsWebHookAllowed`, so can this function be deleted and `IsWebHookAllowed` be renamed to `IsValidWebhook`?

Reply:
@mpierotti-sift   Changes mentioned above are done, Please review them